### ISSUE TITLE
Support for other init system (s6 and runit)

### ISSUE THREAD

**Issue by shoober420** [Nefelim4ag/Ananicy#267]
I’ve recently moved to Artix Linux (s6) and soon learned this awesome program depends on systemd-notify (https://forum.artixlinux.org/index.php/topic,1729.0.html). Is it possible to consider ananicy support other init systems?

**Comment by Nefelim4ag** [comment]
I don't want do that, testing that.
More over AFAIK ananicy don't depends on systemd, so you can write your preffered init file for your init system.

About you forum message, looks like your are new to shell, so:
for rc.local
```
<command> & # `&` fork process to background. `exec` replace your process with new. So looks like you init (pid 1) exec to ananicy and system of course will broke.
```

**Comment by linuxergr** [comment]
@shoober420 I have worked on it for runit on Artix Linux, I hope that the s6 equivalent will also be ready soon.

https://gitea.artixlinux.org/linuxer/PKGBUILDS/src/branch/master/ananicy-runit

**Comment by shoober420** [comment]
@linuxergr 
Awesome, thank you so much. I’ll test it as soon as it’s ready.

**Comment by linuxergr** [comment]
Thank you @shoober420 

Have Artix Linux universe repo enabled at your /etc/pacman.conf, because it will probably go in that repo and there will be no announcement.

Reagrds

**Comment by shoober420** [comment]
@linuxergr 

I currently use s6, so I can test that package.

**Comment by linuxergr** [comment]
@shoober420 the packages are tested and ready for more, at universe, for all Artix Linux init systems.

Have fun!

**Comment by shoober420** [comment]
@linuxergr 

So far its working great. I just installed it and tried out the cmus audio player. I got messages in the terminal about ananicy renicing the process, so everything seems to be running awesome. I'll test out in some games later. Thank you so much!

**Comment by linuxergr** [comment]
@shoober420 correct your (mine) package here https://aur.archlinux.org/packages/ananicy-nosystemd-git/#comment-787322, as I described that you should do, otherwise is ananicy and not ananicy-nosystemd

**Comment by shoober420** [comment]
@linuxergr 
It’s the git version, it works on my machine. Programs I run will have “renice” messages in the terminal.

**Comment by shoober420** [comment]
Here’s a picture of it working on my system while running cmus. Ananicy will display info in the terminal unless disabled.

![502AEBE7-5B9F-4126-A3D0-8935F8186B80](https://user-images.githubusercontent.com/5150898/105252810-33d26280-5b4c-11eb-8fcb-c6a3d8e44de0.jpeg)

**Comment by linuxergr** [comment]
Thanks to my fork of ananicy-nosystemd from Artix and of cource my services at Artix Linux Gitea Repo.

**Comment by shoober420** [comment]
@linuxergr 
I never said I developed anything. Are you saying you are a developer for Ananicy?

**Comment by linuxergr** [comment]
LMAO

**Comment by shoober420** [comment]
@linuxergr 
I just checked, and I ironically have the most contributions to this program, besides the developer of course.

I saw your “fork”. It appears you copied the code base and put some s6 and runit shell scripts in place, as evident in the “ananicy-s6” package I uploaded to the AUR, with the two simple files it installs.

Did you even change any actual code for your ananicy “fork”? If that’s all you’ve done, then we are both a “developer” for ananicy huh?
